pom.xml

```diff
@@ -252,6 +252,7 @@
                 <configuration>
                     <container>
                         <jvmFlags>
+                            <jvmFlag>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005</jvmFlag>
```

Comment:
If this exists it means that every container will have debugging enabled which is not desired for live instances.

I'd rather enable debugging only in Docker env (docker-chs-development repository) by using one of the Java env variables e.g. `JAVA_TOOL_OPTIONS`

Reply:
Fixed this now. This has been added to docker-chs-development here and will only be used locally, https://github.com/companieshouse/docker-chs-development/pull/88/files#diff-a113bffd71a3ef9305be011e8b559f5529b5f555eb3c92ea4fd51202c0145bcfR34